### PR TITLE
Update issue templates to fit Github's data model

### PR DESCRIPTION
**Pull request by EverlastingBugstopper**
Our current issue templates are not being picked up by Github, this PR updates the templates to fit the accepted data model in addition to some style tweaks for the templates themselves.

image

### CODE REVIEW

**Comment by gabbifish** [comment]
Small comments to address, then ready to merge!

**Comment by ashleymichal** [comment]
can you include in the description what "spec" you are following?

**Comment by EverlastingBugstopper** [comment]
Sorry, @gabbifish I'm not seeing your comments?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,36 +1,34 @@
...
-#### What version of Wrangler are you using (`wrangler -V`)?
+  Before opening a new issue, please:
+  * search for existing issues: https://github.com/cloudflare/wrangler/issues
+  * make sure you are using the latest release: https://workers.cloudflare.com/docs/quickstart/updating-the-cli/
+  
+  Thanks! -->
 
-<pre>
-$ wrangler -V
+## Description
```
Comment (gabbifish) [inline .github/ISSUE_TEMPLATE/bug_report.md]: "Description" is not super necessary here; the following fields will gather all the info we need. I'd remove this field.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,36 +1,34 @@
 ---
-title: 'Bug report'
+name: 🐛 Bug report
+about: Something's not quite right? You're in the right place!
 labels: user report
 ---
 
-### 🐛 Bug Report
+# 🐛 Bug Report
```
Comment (gabbifish) [inline .github/ISSUE_TEMPLATE/bug_report.md]: This is going to be massive text! I recommend making this ### and all other fields ####. This applies to the feature request template as well.

**Comment by EverlastingBugstopper** [comment]
Comments addressed!

**Pull request merged**